README.md

```diff
@@ -52,30 +40,89 @@ Ask someone to give you the token when in doubt.
 
 This needs to be done only once.
 
-## Installing / upgrading the package itsef
+## [Requirements](#requirements)
+
+#TODO: write requirements that the web delivery needs to satisfy in order for the module to work
+
+## [How to install](#how-to-install)
+> The module is designed to run in the browser. It can be used in node as well, but the primary target is  for  the browser.
+
+### [Installation via script tag](#install-via-script-tag)
+```html
+<script src="https://unpkg.com/@forward-distribution/bridget"></script>
+<script type="text/javascript">
+  // Initialize the module. Must be called before any other methods
+  bridget.init();
```

Comment:
Maybe I'm missing an implementation details, but I think we can do the init when we import the library, right? And trough a ready event when the initialisation is done. If we have the event, we can do an API or wrap over it, that check that the even has been fired.

Not sure if you understand me, but in essence, we can have a Promise that can be used to check if we are ready or not:
```javascript
export const fpBridgeReady = new Promise((resolve, reject) => {
  if (inWebView) {
    document.addEventListener('@fp/bridgetReady', () => resolve(true));
  } else {
    reject(true); // if not on webview, reject
  }
});
```

Reply:
No implementation details, everything is up for discussion :) 
That `init()` is just a placeholder. In essence, yes, I agree with you that we want to do the init internally when the library is loaded.  Let's follow up on this.